src/mpid/ch4/src/ch4r_win.c

```diff
@@ -562,7 +562,7 @@ static int win_shm_alloc_impl(MPI_Aint size, int disp_unit, MPIR_Comm * comm_ptr
                 shared_table[i].shm_base_addr = NULL;
 
             if (MPIDIG_WIN(win, info_args).alloc_shared_noncontig)
-                cur_base += MPIDIU_get_mapsize(shared_table[i].size, &page_sz);
+                cur_base += MPIDU_shm_get_mapsize(shared_table[i].size, &page_sz);
             else
```

Comment:
You already replaced `MPIDIU_get_shm_symheap` in the previous commit. Actually I think the replace|remove commits can be merged into a single one.